packages/wearable_rotary/README.md

```diff
@@ -0,0 +1,36 @@
+# wearable_rotary
+
+Plugin that can listen to rotary events on Galaxy watch devices.
+
+## Usage
+
+To use this plugin, add wearable_rotary as a dependency in your pubspec.yaml file:
+
+```yaml
+dependencies:
+  wearable_rotary: ^1.0.0
+```
+
+## Example
+
+```dart
+// Import package
+import 'package:wearable_rotary/wearable_rotary.dart';
+
+// Be informed when the event (RotaryEvent.CLOCKWISE, RotaryEvent.COUNTER_CLOCKWISE) occurs
```

Comment:
Please update the comment (to lowercase).

Also please consider ending your comment with a period.

https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences